### ISSUE TITLE
Spy logging interferes with compiled JS in CLJS

### ISSUE THREAD

**Issue by mrrodriguez** [vvvvalvalval/scope-capture#24]
I still get logger output interleaved with the JS output that was discussed in #1 
when using `[vvvvalvalval/scope-capture "0.2.0"]` in ClojureScript with Leiningen/cljs-build/figwheel setup.

This problem seems to come and go from project to project or release to release of `scope-capture` for me. I haven't been able to narrow it down to a basic new project to reproduce it. I think it may be something with the combination of the tools involved. 

I'll add that I used to be able to use `scope-capture` with this same setup around the 2nd release. I am not sure I have enough details to be actionable here. I'm curious if anyone else is facing similar.

**Comment by vvvvalvalval** [comment]
@mrrodriguez I suspect this only happens when cold-compiling a codebase that already has spy statements in it, can you confirm?

**Comment by nickstares** [comment]
I am also having this same issue, with the same figwheel and lein versions that @mrrodriguez is using.

**Comment by nickstares** [comment]
Any progress on this? Debugging the analyzer is a bit beyond my abilities, but happy to help with evidence collection etc.

**Comment by vvvvalvalval** [comment]
@nickstares yes I'd need a precise repro to make progress please - a minimal project reproducing the problem with the exact dependencies would be ideal.

**Comment by mrrodriguez** [comment]
@vvvvalvalval I may try to get a reproducing case again today. I've had a hard time doing so before. When I make a small, dummy project, with a bunch of the same core dev dependencies, it hasn't happened. I have 2 "real" separate projects though, where it happens the same in each. I've been completely unable to use `scope-capture` in my CLJS-side due to that. 
So since it doesn't happen easily in a repro-case, the problem becomes trying to have a clue what is happening in the analyzer to try to recreate it...
Hopefully I can figure something out. I know there isn't much for you to go on from this issue so far.

**Comment by mrrodriguez** [comment]
@nickstares I was able to recreate it in a similar way and agree that the problem only seems to be with the `figwheel-sidecare` + `piggieback` + `cider` setup.

I'm thinking `cider` doesn't matter, it's the other 2 involved. So will see about being able to narrow it down more.

**Comment by SevereOverfl0w** [comment]
The cljs compiler uses `*out*` for emitting strings.  If you instead defaulted to `*err*`, then it wouldn't conflict.
